### PR TITLE
Fixes and setup for babel

### PR DESCRIPTION
**Pull request by ohler55**
This fixes the embedded conf file that was not updated when the sample comtroller
class was changed. It also sets up for use of babel in the future.

### CODE REVIEW

File: examples/sample/opo/embed.conf [hunk 1]
```diff
@@ -237,7 +237,7 @@ handler.embed.class = SampleController
 # support calling the embedded interpreter.
 
 embed.load_path = .,lib,examples/sample
-embed.require = sample
+embed.require = sample_controller
```
Comment (ashmaroli) [inline examples/sample/opo/embed.conf]: you could have pushed this directly to the `develop` branch..

File: package.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "name": "wabur",
+    "version": "0.2.0",
+    "scripts": {
+        "build": "babel -o view/pages/assets/js/wab.js view/js/wab.js view/js/list.js view/js/obj.js view/js/view.js"
```
Comment (ashmaroli) [inline package.json]: haven't tried this syntax yet.. but there's an `--out-dir` switch and `-w` to compile an entire directory and watch it for any changes..
perhaps the order is necessary to compile the JS in that order?

File: notes [hunk 3]
```diff
@@ -1,2 +1,13 @@
 ;; -*- mode: outline; outline-regexp: " *[-\+]"; indent-tabs-mode: nil -*-
 
+- installing npm and babel
+ - ./node_modules/.bin/babel -o view/pages/assets/js/wab.js view/js/wab.js view/js/list.js view/js/obj.js view/js/view.js"
```
Comment (ashmaroli) [inline notes]: consider using `--out-dir` or `-d` and `-w` along with `--presets es2015` when we make the switch to ES6 modules

File: examples/sample/opo/embed.conf (hunk 1 above)
Comment (ohler55) [reply examples/sample/opo/embed.conf]: I thought about it but figured I'd give you a chance to review.

File: package.json (hunk 2 above)
Comment (ohler55) [reply package.json]: The order was necessary to get it to work. wab.js had to be first which was not the default.

File: notes (hunk 3 above)
Comment (ohler55) [reply notes]: `-o` and --out-dir` are the same. `--minify` is something we should add as well once we get the new code debugged.

Comment (ashmaroli) [reply notes]: they're not the same.. please check `babel --help` once more:
```
    -o, --out-file [out]                 Compile all input files into a single file
    -d, --out-dir [out]                  Compile an input directory of modules into an output directory
```

Comment (ashmaroli) [reply notes]: But **currently** in our case, `-o` is the right choice. Once we make a proper move to ES6 modules, we can switch to the directory option

Comment (ohler55) [reply notes]: You are correct although `-o` or `--out-file` is what we want.

**Comment by ashmaroli** [comment]
with this, the `build-view` ruby script would need to be edited as well..

File: script/build-view [hunk 4]
```diff
@@ -3,132 +3,47 @@
...
-    %w(
-      wab
-      view
-      list
-      obj
-    ).each { |src| compact_js("#{src}.js", file) }
+def generate_js
+  minified = ARGV.include?('-m') ? '--minified ' : ''
+  dest = "#{dest_dir('js')}/wab.js"
+  out = `#{path_to('node_modules', '.bin', 'babel')} #{minified}-o #{dest} #{js_src_dir}/wab.js #{js_src_dir}/list.js #{js_src_dir}/obj.js #{js_src_dir}/view.js`
```
Comment (ashmaroli) [inline script/build-view]: why not just rely on the `package.json` and shell out to `npm`?

Comment (ohler55) [reply script/build-view]: I didn't see any advantage to the extra wrapper. This is more direct.

**Pull request merged**